docs/index.md

```diff
@@ -64,9 +64,36 @@ The current version cover the following elements:
 
 Jump to [Syntax](syntax.md).
 
-## Fast.match?
+## ast
 
-`match?` is the most granular function that tries to compare a node with an
+Use `Fast.ast` to convert simple code to AST objects. You can use it as
+`ruby-parse` but directly from the console.
+
+```ruby
+Fast.ast("1")       # => s(:int, 1)
+Fast.ast("method")  # => s(:send, nil, :method)
+Fast.ast("a.b")     # => s(:send, s(:send, nil, :a), :b)
+Fast.ast("1 + 1")   # => s(:send, s(:int, 1), :+, s(:int, 1))
+Fast.ast("a = 2")   # => s(:lvasgn, :a, s(:int, 2))
+Fast.ast("b += 2")  # => s(:op_asgn, s(:lvasgn, :b), :+, s(:int, 2))
+```
+
+It uses [astrolable](https://github.com/yujinakayama/astrolabe) gem behind the scenes:
+
+```ruby
+Fast.ast(Fast.ast("1").class
+=> Astrolabe::Node
+Fast.ast(Fast.ast("1").type
+=> :int
+Fast.ast(Fast.ast("1").children
+=> [1]
```

Comment:
Missing closing parens on all three of these after `("1")` I think

Reply:
thanks a lot! something went wrong while copy and pasting 😄